src/utils/list_cpu_features.c

```diff
@@ -408,6 +413,12 @@ static Node* CreateTree(void) {
   AddMapEntry(root, "microarchitecture",
               CreateString(strings.type.base_platform));
   AddFlags(root, &info.features);
+#elif defined(CPU_FEATURES_ARCH_RISCV)
+  const RiscvInfo info = GetRiscvInfo();
+  AddMapEntry(root, "arch", CreateString("risc-v"));
```

Comment:
With enough time people will rely on this string. Are we sure we want "risc-v"?
Alternative naming scheme I can think of :
 - "riscv"
 - "riscv-32"/ "riscv-64" / "riscv-128"

Reply:
the second naming scheme seems better to me since this functionality is implemented with `GetRiscvInfo` and more precise arch detection compared to riscv, but I also don't mind `"riscv"`

Reply:
As an afterthought, it seems that the 32 / 64 / 128 address space is really seen as a feature so we could stick to "risc-v". As you said the information exists elsewhere. I don't mind either, I just wanted to check whether we could regret this choice later on.